data/json/bionics.json

```diff
@@ -859,9 +859,10 @@
     "id": "bio_solar",
     "type": "bionic",
     "name": "Solar Panels",
-    "description": "You have a few solar panels installed.  While in direct sunlight, your power level will slowly recharge.",
+    "description": "You have small solar panels contained in a box on your back.  On activation they will retract in a butterfly wings manner, and while in direct sunlight, your power level will slowly recharge.",
     "occupied_bodyparts": [ [ "TORSO", 20 ] ],
-    "flags": [ "BIONIC_POWER_SOURCE" ]
+    "flags": [ "BIONIC_POWER_SOURCE", "BIONIC_TOGGLED" ],
+    "act_cost": 10
```

Comment:
How this balanced against to produced amount of power? Did you test this? Do you have any assumption how this should be balanced ("activation energy will be restored by 1hr on sun" or something in this way?)
Why there is no ```deact_cost```?

Reply:
I wasn't even aware of deactivation cost. Added.

I tested only in bright lighting. The produced power is 25, so it's 2,5 times the activation cost. Honestly I think that's too much for a one-time power add.